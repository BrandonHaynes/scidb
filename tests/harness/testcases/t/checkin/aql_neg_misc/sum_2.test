# http://trac.scidb.org/ticket/325#comment:18

--setup
--aql create array A <a:int32>[x=0:2,3,0]
store(build(A,x+1),A)

--test
--error --code=scidb::SCIDB_SE_SYNTAX::SCIDB_LE_ATTRIBUTE_NOT_EXIST --aql="select sum(A) from A group by x"


--cleanup
--aql drop array A
