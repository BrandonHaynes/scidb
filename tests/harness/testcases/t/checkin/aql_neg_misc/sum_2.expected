Query was executed successfully

{x} a
{0} 1
{1} 2
{2} 3

[An error expected at this place for the query "select sum(A) from A group by x". And it failed with error code = scidb::SCIDB_SE_SYNTAX::SCIDB_LE_ATTRIBUTE_NOT_EXIST. Expected error code = scidb::SCIDB_SE_SYNTAX::SCIDB_LE_ATTRIBUTE_NOT_EXIST.]

Query was executed successfully

